draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -374,15 +391,33 @@ serial.<unique-2>.zones.$CATZ 0 IN SERIAL 1634730530
 serial.<unique-3>.zones.$CATZ 0 IN SERIAL 2020112405
 ```
 
-## Custom properties {#customproperties}
-
-More properties may be defined in future documents.
-These future properties will be represented by RRsets directly below the name of a member node.
+## Custom Properties (`*.ext` properties) {#customproperties}
 
 Implementations and operators of catalog zones may choose to provide their own properties.
-To prevent a name clash with future properties, private properties should be represented below the label `ext.<unique-N>.zones.$CATZ`.
-`ext` is not a
-placeholder, so a custom property would have the domain name `<your-label>.ext.<unique-N>.zones.$CATZ`
+Custom properties can occur both globally, or for a specific member zone.
+To prevent a name clash with future properties, such properties should be represented below the label `ext`.
+
+`ext` is not a placeholder, so a custom property would have domains names as follows:
+
+```
+<your-property>.ext.$CATZ                   # for a global custom property
+<your-property>.ext.<unique-N>.zones.$CATZ  # for a member zone custom property
```

Comment:
Suggestion:
```ext.<unique-N>.zones.$CATZ                                   # will clear al defined global custom properties for <unique-N>```

Reply:
What content does one put in `ext....` to clear?

Reply:
I would prefer to leave that out in this draft to be honest. `.ext` options are implementation specific anyway. If an implementation defines an option it can also define how to clear it.

Reply:
I agree, and pt01, at my request, did an update last week to make clear that global properties are not member-default properties unless their definition says they are.

Reply:
Is there also a pt02?

Reply:
Sorry, I know @peterthomassen as pt01 :)